translation from c00/30/16 to 20 (#66)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/15/2/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//
自分の部屋より、あなたのお部屋の方が落ち着くんです。……これからも、私を傍においていただけますか？=I feel more comfortable in your room than mine... Is it ok if I stay with you more from now on?
//general purpose:

> 女が好きな方の元へ訪れたのですから、セックスをするのは自明のことですね。=Since you're known for liking women, it's obvious that I came to have sex. 

Reviewer: ```suggestion
女が好きな方の元へ訪れたのですから、セックスをするのは自明のことですね。=Since you're known to like women, it's obvious that I came to have sex. 
```

このまま一日中情事に耽りたいのですが……我儘を言ってはいけませんね。=I'd like to indulge in an all-day affair like this, but... I shouldn't be selfish.

> ありがとうございました。これ以上お傍にいたら、益々離れがたくなるので、これで失礼致します。=Thank you so much! The longer I stay with you, the harder, the harder it is to part, so I will take my leave now.

Reviewer 2: Some text got duplicated
```suggestion
ありがとうございました。これ以上お傍にいたら、益々離れがたくなるので、これで失礼致します。=Thank you so much! The longer I stay with you, the harder it is to part, so I will take my leave now.
```

> ありがとうございました。これ以上お傍にいたら、益々離れがたくなるので、これで失礼致します。=Thank you so much! If I stay with you any longer, it will be much more dificult to part, so I will take my leave now.

Reviewer 2: Simpler?
```suggestion
ありがとうございました。これ以上お傍にいたら、益々離れがたくなるので、これで失礼致します。=Thank you so much! If I stay with you any longer, it will be hard to leave, so I'll go now.
```

Or less literal?
```suggestion
ありがとうございました。これ以上お傍にいたら、益々離れがたくなるので、これで失礼致します。=Thank you so much! The longer I stay with you, the harder it is to leave, so I'd better go now.
```

Author: i like the 2nd one more, so i went with it. considered using a but in front, decided against it.
